### PR TITLE
[1LP][RFR]Implemented automated test for test_deploy_instance_with_ssh_addition_template

### PR DESCRIPTION
**Pull request by mmojzis**
{{ pytest: -k "test_deploy_instance_with_ssh_addition_template" -v }}

### CODE REVIEW

**Comment by dajoRH** [comment]
I detected some fixture changes in commit a326f7bf4fffa701bf95a400597c45fdc2af68dc

The *local* fixture **`instance_with_ssh_addition_template`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

File: cfme/tests/cloud/test_providers.py [hunk 1]
```diff
@@ -188,6 +188,16 @@ def vm_ip(cfme_vhd, pwsh_ssh):
         pwsh_ssh.run_command("pwsh {}".format(path_script), timeout=180)
 
 
+@pytest.fixture()
```
Comment (john-dupuy) [inline cfme/tests/cloud/test_providers.py]: _Optional_ `()` not necessary

Comment (mmojzis) [reply cfme/tests/cloud/test_providers.py]: Fixed

**Comment by dajoRH** [comment]
I detected some fixture changes in commit e13cf5e7c28f4a70b399cc6f537fafc2fcc93df6

The *local* fixture **`instance_with_ssh_addition_template`** is used in the following files:
- **cfme/tests/cloud/test_providers.py**

Please, consider creating a PRT run to make sure your fixture changes do not break existing usage :smiley:

**Pull request merged**